src/sidebar/components/annotation-omega.js

```diff
@@ -79,14 +62,7 @@ function AnnotationOmega({
     createDraft(annotation, { ...draft, text });
   };
 
-  const onReply = () => {
-    // TODO: Re-evaluate error handling here
```

Comment:
Error-handling is now handled within the `AnnotationActionBar` component where this is wrapped. In fact, really, the whole shebang should move into `AnnotationActionBar` once we stop using the legacy `Annotation` component, as there's no need for `AnnotationOmega` to be involved in reply creation.